app/src/main/java/app/revanced/integrations/sponsorblock/requests/SBRequester.java

```diff
@@ -178,7 +188,8 @@ public static void retrieveUserStats(PreferenceCategory category, Preference loa
     public static void setUsername(String username, EditTextPreference preference, Runnable toastRunnable) {
         new Thread(() -> {
             try {
-                HttpURLConnection connection = getConnectionFromRoute(SBRoutes.CHANGE_USERNAME, SettingsEnum.SB_UUID.getString(), username);
+                HttpURLConnection connection = getConnectionFromRoute(SBRoutes.CHANGE_USERNAME,
```

Comment:
Theres lots of unrelated refactoring changes, please separate refactoring as a different commit

Reply:
problem is my auto template save. i will correct it.